doc/website/release-notes/iceoryx-v2-0-0.md

```diff
@@ -83,6 +83,7 @@ fb913bf0de288ba84fe98f7a23d35edfdb22381
 - Prevent creation of `popo::Publisher`'s with internal `ServiceDescription` [\#1120](https://github.com/eclipse-iceoryx/iceoryx/issues/1120)
 - RelativePointer is now type safe, i.e. can only be constructed from pointers with a valid convertion to the raw pointer [\#1121](https://github.com/eclipse-iceoryx/iceoryx/issues/1121)
 - Clamping `historyRequest` to `queueCapacity` [\#1192](https://github.com/eclipse-iceoryx/iceoryx/issues/1192)
+- C binding storage sizes do not match for multiple OS's and architectures [\#1218](https://github.com/eclipse-iceoryx/iceoryx/issues/1218)
```

Comment:
Please describe the fix, not the bug.

Reply:
According to @dkroenke we can also use the description of the issue. This is also more helpful for the user since they know the bug bug and might not be able to connect the fix to the bug

Reply:
That's okay with me, but we provide the link for the connection...